src/bin/test_userproc_even.c

```diff
@@ -0,0 +1,29 @@
+int main();
+int _start() {
+    return main();
+
+    // invoke system call that kills the process
+}
+
+#include "../syscall.h"
```

Comment:
Should we be moving the executable files one directory up or something? This seems weird